app/Providers/RouteServiceProvider.php

```diff
@@ -192,6 +193,22 @@ protected function mapWebRoutes(Router $router)
         });
     }
 
+    /**
+     * Define the custom oauth routes for the API.
+     *
+     * @param  \Illuminate\Routing\Router  $router
+     * @return void
+     */
+    protected function mapOAuthRoutes(Router $router)
+    {
+        $router->group([
+            'prefix' => 'oauth',
+            'namespace' => $this->namespace,
+        ], function ($router) {
+            require base_path('routes/oauth.php');
+        });
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the unused function parameter "$router". ([php:S1172](https://sonarcloud.io/organizations/monicahq/rules#rule_key=php%3AS1172))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=monica&pullRequest=1174&issues=AWLRaBe7-pl6AHs2IiNi&open=AWLRaBe7-pl6AHs2IiNi)

[ISSUE_KEY]: AWLRaBe7-pl6AHs2IiNi